<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageShowResult</class>
 <widget class="QMainWindow" name="pageShowResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(253, 253, 253);
}
QToolBar{
	border-top:1px solid gray;
	border-bottom:1px solid gray;
}
QAction{
	background-color: rgb(250, 250, 250);
	border:1px solid gray; border-radius:5px;
}
QAction:hover{
	background-color: rgb(245, 245, 245);
	border:1px solid gray; border-radius:5px;
}
QAction:pressed{
	background-color: rgb(240, 240, 240);
	border:1px solid gray; border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="overlappedIma">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageShowDIFF">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="before">
              <property name="text">
               <string>Before</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="after">
              <property name="text">
               <string>After</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageAlterStyle">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QRadioButton" name="altCub">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Cubes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="altCirc">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Circles</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageFeedback">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Question1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QRadioButton" name="radioButton_3">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_4">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_5">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_6">
                   <property name="text">
                    <string>RadioButton</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Question2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>461</width>
                 <height>61</height>
                </rect>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>60</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actShowDIFF"/>
   <addaction name="actChangeStyle"/>
   <addaction name="actFeedback"/>
   <addaction name="actDownload"/>
  </widget>
  <action name="actShowDIFF">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/diff.png</normaloff>:/images/images/diff.png</iconset>
   </property>
   <property name="text">
    <string>Show DIFF</string>
   </property>
   <property name="toolTip">
    <string>Show both before and after overlapped image</string>
   </property>
  </action>
  <action name="actChangeStyle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/alter.png</normaloff>:/images/images/alter.png</iconset>
   </property>
   <property name="text">
    <string>Alter Style</string>
   </property>
   <property name="toolTip">
    <string>Choose one overlap style</string>
   </property>
  </action>
  <action name="actFeedback">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/feedback.png</normaloff>:/images/images/feedback.png</iconset>
   </property>
   <property name="text">
    <string>Feedback</string>
   </property>
   <property name="toolTip">
    <string>Give feedback to producers</string>
   </property>
  </action>
  <action name="actDownload">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/download.png</normaloff>:/images/images/download.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download overlapped image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
